src/Platforms/OraclePlatform.php

```diff
@@ -593,6 +610,24 @@ private function normalizeIdentifier($name): Identifier
         return $identifier->isQuoted() ? $identifier : new Identifier(strtoupper($name));
     }
 
+    /**
+     * Quotes a string if it was also initially quoted.
+     *
+     * @param string $name The identifier name.
+     *
+     * @return string The quoted identifier string if necessary.
+     */
+    private function getOriginalIdentifier($name)
```

Comment:
This looks a bit shady. If the identifier was originally quoted, why does the platform need to guess it? The existing API allows passing quoted identifiers. Why wouldn't the caller pass it explicitly quoted if it has to be quoted?